components/eamxx/src/physics/share/scream_trcmix.cpp

```diff
@@ -14,6 +14,7 @@ namespace physics {
 
 void trcmix(
   const std::string& name,
+  const int nlevs,
```

Comment:
If we are going to pass nlevs, why not also pass ncols for consistency?

Reply:
`ncols` can safely be inferred from the view dimensions. `nlevs` on the other hand cannot be inferred, since we don't know if the views are padded or not.